src/main/kotlin/pt/up/fe/ni/website/backend/service/AccountService.kt

```diff
@@ -26,4 +27,15 @@ class AccountService(private val repository: AccountRepository, private val enco
 
     fun getAccountByEmail(email: String): Account = repository.findByEmail(email)
         ?: throw NoSuchElementException(ErrorMessages.emailNotFound(email))
+
+    fun changePassword(id: Long, dto: ChangePassDto): Map<String, String> {
```

Comment:
Does this need to return a `Map`?

Reply:
Yes, otherwise the response is a 500 error saying "No acceptable representation". I put it with an empty map for consistency like others class do.